global_configs.exs

```diff
@@ -0,0 +1,29 @@
+use Mix.Config
+
```

Comment:
Having one global config is usually pretty nasty. It's usually better to explicitly configure each app individually. 
You also don't really want to be referencing modules that don't exist in a given env. This is usually handled by doing something like:

`import_config("#{Mix.env()}.exs)`
where you define a `test.exs` and a `dev.exs` etc. 

Also somewhat personal preference, but i don't usually like asigning vars and having maps, variables or function calls in config.exs entries. These configs are cached, so there is the posibility of storing a term that isn't persisted properly between VM exits.